kernel/thread.c

```diff
@@ -680,6 +687,13 @@ static long _syscall_clone_vfork(
         child->run_thread = myst_run_thread;
         child->main.thread_group_lock = MYST_SPINLOCK_INITIALIZER;
         child->thread_lock = &child->main.thread_group_lock;
+        {
+            char buf[16];
+            /* ATTN: we don't take a lock on _num_threads,
+             thread names could be duplicates */
+            snprintf(buf, sizeof(buf), "thread-%ld", _num_threads % 99999999);
```

Comment:
Actually, I meant to replace the whole block with one line:
```
snprintf(child->name, sizeof(buf), "thread-%ld", _num_threads % 99999999);
```

Reply:
Oh right, thats much simpler 👍

Reply:
Fixed.